### DISCUSSION TITLE
Possiblity to set base Branch of Pipeline run as target Branch for

### DISCUSSION THREAD

**Discussion by schamal2205** [renovatebot/renovate#18043] [Request Help] [answered]
E.g. if Renovate Pipeline ist running on brach "test" i want the Pullrequest to be created with target-branch "test" but Pullrequest ist created with target-branch "master".
Is there a way to tell renovate during runtime to create PR against the base-branch instead of using the baseBranches definition in renovate.json?

**Comment by rarkins** [comment, marked as answer]
Renovate will always read config from the default branch of a repo.

But you could also run it dynamically specifying baseBranches through env, e.g. 

```
RENOVATE_BASE_BRANCHES=`git branch --show-current` renovate
```
